internal/local/tls.go

```diff
@@ -16,42 +17,138 @@ const RemoteTLSInstanceId = `Intel(r) AMT 802.3 TLS Settings`
 const LocalTLSInstanceId = `Intel(r) AMT LMS TLS Settings`
 
 func (service *ProvisioningService) ConfigureTLS() error {
-	log.Info("configuring TLS")
+	var err error
+	if service.flags.ConfigTLSInfo.EAAddress != "" && service.flags.ConfigTLSInfo.EAUsername != "" && service.flags.ConfigTLSInfo.EAPassword != "" {
```

Comment:
Use the url package to parse the url and ensure it is valid.  You'll likely need to add a flag to allow skipping cert check if the server is using self signed with https://

Reply:
Parsing is complete. A flag to allow skipping certificate checks will be added in a new pull request as soon as it is ready.